internal/pkg/metadb/metadb.go

```diff
@@ -898,8 +898,8 @@ func (m *MetaDB) GetMultiRecords(ctx context.Context, storeKeys, recordKeys []st
 	records := make([]*record.Record, len(keys))
 	if err = m.client.GetMulti(ctx, keys, records); err != nil {
 		if _, ok := err.(ds.MultiError); ok {
-			// Error(s) encountered when getting some entities
-			return records, err
+			// Error(s) encountered when getting some entities, ignore
+			return records, nil
```

Comment:
Is there a reason we're ignoring these errors?

Reply:
I went back and forth on whether or not returning these errors was actually useful for the caller. And how to actually get the results back to the Open Saves client.
I will work on another revision where I'll be returning either a record or the error to keep the index consistent and inform the open saves client that something happened.